index.bs

```diff
@@ -3026,7 +2986,6 @@ optionally evidence of [=user consent=] to a specific transaction.
         [SameObject] readonly attribute ArrayBuffer      authenticatorData;
         [SameObject] readonly attribute ArrayBuffer      signature;
         [SameObject] readonly attribute ArrayBuffer?     userHandle;
-        [SameObject] readonly attribute ArrayBuffer?     attestationObject;
```

Comment:
This will need to be dropped from `AuthenticatorAssertionResponseJSON` as well (it was added there too back in #1920)